File: source/kCura.Relativity.Export.ConvertedToC#/FileObjects/ErrorExportFile.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Linq;
namespace kCura.Relativity.Export.FileObjects
{

	public class ErrorExportFile : ExportFile
	{
		private string _errorMessage = string.Empty;
		public ErrorExportFile(string errorMessage) : base(-1)
		{
			if (string.IsNullOrEmpty(errorMessage))
				throw new System.ArgumentException("Error message cannot be null for an error export file");
			_errorMessage = errorMessage;
		}
		public string ErrorMessage {
			get { return _errorMessage; }
		}
	}
}

File: source/kCura.Relativity.Export/FileObjects/ErrorExportFile.vb
Namespace kCura.Relativity.Export.FileObjects

	Public Class ErrorExportFile
		Inherits ExportFile
		Private _errorMessage As String = String.Empty
		Public Sub New(ByVal errorMessage As String)
			MyBase.New(-1)
			If String.IsNullOrEmpty(errorMessage) Then Throw New System.ArgumentException("Error message cannot be null for an error export file")
			_errorMessage = errorMessage
		End Sub
		Public ReadOnly Property ErrorMessage As String
			Get
				Return _errorMessage
			End Get
		End Property
	End Class
End Namespace
